genie-common/src/main/java/com/netflix/genie/common/exceptions/GenieUserLimitExceededException.java

```diff
@@ -0,0 +1,73 @@
+/*
+ *
+ *  Copyright 2015 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.common.exceptions;
+
+/**
+ * Extension of a GenieException for a user exceeding some limit (e.g., submitting too many jobs).
+ *
+ * @author mprimi
+ * @since 3.1.0
+ */
+public final class GenieUserLimitExceededException extends GenieException {
+
+    private static final String ACTIVE_JOBS_LIMIT = "activeJobs";
+    private final String user;
+    private final String exceededLimitName;
+
+    private GenieUserLimitExceededException(
+        final String user,
+        final String limitName,
+        final String message
+    ) {
+        super(429, message);
+        this.user = user;
+        this.exceededLimitName = limitName;
+    }
+
+    public String getUser() {
```

Comment:
just add `@Getter` from lombok to the class so that you don't have to define this boilerplate yourself.